features/step_definitions/rails_steps.rb

```diff
@@ -1,5 +1,14 @@
 When /^I create a new rails application$/ do
-  options = "--skip-bootsnap --skip-javascript"
+  options =
+    %w[
+      --api
```

Comment:
It looks like [`--api` automatically takes care of `--skip-javascript` and `--skip-sprockets`](https://github.com/rails/rails/blob/157920aead96865e3135f496c09ace607d5620dc/railties/lib/rails/generators/rails/app/app_generator.rb#L277-L279), but I sort of like keeping those options here anyway, to make it more explicit what we are trying to leave out. 👍